### ISSUE TITLE
Just listing the things i sent

### ISSUE THREAD

**Issue by Welsh-Boogie** [sagishporer/hashcode-2021-qualification#9]
i sent alot of things so hope u received them

1. hashcode notes
2. warsaw husks notes
3. code
4. an idea or 2

**Comment by sagishporer** [comment]
Yes, however, I've switched to brute force search. I hope it will work.
